ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -54,24 +60,17 @@ import { getNetworkLabelKey } from '../../../../helpers/utils/i18n-helper';
  * it can't be converted.
  */
 const getDisplayChainId = (chainId) => {
```

Comment:
this is funky now. 

user input gets transformed into CaipChainId from either decimal or hex formatted chain id, but when displayed again to the user after being saved, it's shown in decimal since we don't know if we should go from CaipChainId to decimal or hex anymore